components/sup/src/manager/census.rs

```diff
@@ -14,37 +14,37 @@
 
 use std::ops::{Deref, DerefMut};
 use std::collections::HashMap;
-use std::str::FromStr;
+use std::str::{self, FromStr};
 
 use hcore::package::PackageIdent;
 use hcore::service::ServiceGroup;
-use butterfly::rumor::service::Service as ServiceRumor;
-use butterfly::rumor::election::{Election as ElectionRumor, Election_Status};
+use butterfly;
+use butterfly::rumor::{Service as ServiceRumor, Election as ElectionRumor};
```

Comment:
Yeah, I see those imports coming back. I think this is worse, not better, even though the above is more terse. It maps to the actual locations on disk - with this indirection, you have to go code spelunking, discover the re-export, and *then* find it on disk. I'm fine if its the pattern we want to follow, but I'm :-1: personally.